### PR TITLE
Security vulnerabilities in 3rd party jars

### PR DESCRIPTION
**Pull request by rajsarkapally**

### CODE REVIEW

File: ArgusCore/pom.xml [hunk 1]
```diff
@@ -341,7 +341,7 @@
         <dependency>
             <groupId>org.apache.hbase</groupId>
             <artifactId>hbase-client</artifactId>
-            <version>0.98.8-hadoop2</version>
+                <version>1.4.2</version>
```
Comment (dilipdevaraj-sfdc) [inline ArgusCore/pom.xml]: This should eba . No-Op in testing sicne we are not using AsynHBaseScehmaService

File: ArgusSDK/pom.xml [hunk 2]
```diff
@@ -111,6 +111,43 @@
                 <filtering>true</filtering>
             </testResource>
         </testResources>
+        <pluginManagement>
+        	<plugins>
+        		<!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself.-->
```
Comment (dilipdevaraj-sfdc) [inline ArgusSDK/pom.xml]: Is this change intended to be checked in ?
Looks like it is Eclipse specifc setting.

File: pom.xml [hunk 3]
```diff
@@ -118,9 +118,9 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
         <dockerImages.base.name>salesforce</dockerImages.base.name>
-        <project.version>2.12.0</project.version>
+        <project.version>2.12.1</project.version>
         <logback.version>1.2.3</logback.version>
-        <jackson.version>2.9.2</jackson.version>
+        <jackson.version>2.9.5</jackson.version>
```
Comment (dilipdevaraj-sfdc) [inline pom.xml]: The jackson is used for serialization/deserialization.

Do we have any instance where some specific components were not functioning correctly due to this version upgrade ?

File: pom.xml [hunk 4]
```diff
@@ -118,9 +118,9 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
         <dockerImages.base.name>salesforce</dockerImages.base.name>
-        <project.version>2.12.0</project.version>
+        <project.version>2.12.1</project.version>
```
Comment (dilipdevaraj-sfdc) [inline pom.xml]: Since we released 2.13.0, this version should be updated.

**Comment by rajsarkapally** [comment]
can one of you review it so that i will cut the branch today

@dilip-devaraj @sundeepsf @kgowdru

File: pom.xml [hunk 5]
```diff
@@ -118,9 +118,9 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
         <dockerImages.base.name>salesforce</dockerImages.base.name>
-        <project.version>2.12.0</project.version>
+        <project.version>2.13.0</project.version>
```
Comment (dilipdevaraj-sfdc) [inline pom.xml]: @rajsarkapally 
The new Argus version going out will be 2.14.0.  Can you change this in project.version.

File: ArgusCore/src/main/java/com/salesforce/dva/argus/service/tsdb/AbstractTSDBService.java [hunk 6]
```diff
@@ -833,5 +840,28 @@ public Properties getServiceProperties() {
 	public List<Annotation> getAnnotations(List<AnnotationQuery> queries) {
 		throw new UnsupportedOperationException("This method should be overriden by a specific implementation.");
 	}
+	/**
+	 * Used to close http connections after reusing the same connection for certain number of times 
+	 * @author rsarkapally
```
Comment (dilipdevaraj-sfdc) [inline ArgusCore/src/main/java/com/salesforce/dva/argus/service/tsdb/AbstractTSDBService.java]: @rajsarkapally 
Can you send VIP HTTP changes as a separate PR, so only "Security vulnerabilities in 3rd party jars" is included in this PR for approval.
